nixpkgs-pinned.nix

```diff
@@ -1,4 +1,8 @@
-fetchTarball {
-  url = "https://github.com/NixOS/nixpkgs/archive/b0bbacb52134a7e731e549f4c0a7a2a39ca6b481.tar.gz";
-  sha256 = "15ix4spjpdm6wni28camzjsmhz0gzk3cxhpsk035952plwdxhb67";
-}
+let
```

Comment:
Could you turn this nix file into a function? I would expect `import ./nixpkgs-pinned.nix` to return a lambda. You can add arguments to make the `config` and `overlays` configurable:

```nix
{ overlays ? [] # additional overlays
, config ? {} # Imported configuration
}:
```

The `config` can then be imported as such:

```nix
config = {
      imports = [ config ];
};
```

Reply:
Thanks for a suggestion - what is it for?

Reply:
Oh, one more thing. I figured that you need this for configurability @FPtje . Please note that the whole nix stuff is used here only for packaging the app. Normal development does not warrant Nix.

Reply:
It is mostly for consistency with our other repositories. The expectation is that an import of a nixpkgs file is a function. The extra configuration is not currently needed.

Reply:
Makes sense - will fix

Reply:
Fixed in 155953e